service/http.balancer/src/test/java/org/kaazing/gateway/service/http/balancer/WsxBalancerIT.java

```diff
@@ -54,7 +53,6 @@
                             .allowOrigin("*")
```

Comment:
I believe 3 of those 4 tests already exist. 
- `WSX with kl=y (sends balancer bytes)` - [legacy.wsx.balancer.request](https://github.com/kaazing/gateway/blob/develop/service/http.balancer/src/test/scripts/org/kaazing/gateway/service/http/balancer/legacy.wsx.balancer.request.rpt)
- `WSX without kl=y ( sends wrapped 302)` - [wsx.balancer.request](https://github.com/kaazing/gateway/blob/develop/service/http.balancer/src/test/scripts/org/kaazing/gateway/service/http/balancer/wsx.balancer.request.rpt) which is modified in this PR, [here](https://github.com/kaazing/gateway/pull/870/files#diff-a56af8bf0eb96b06a70a193c8596adf6)
- `WS without kl=y (sends 302)` - [wsn.balancer.request](https://github.com/kaazing/gateway/blob/develop/service/http.balancer/src/test/scripts/org/kaazing/gateway/service/http/balancer/wsn.balancer.request.rpt)

So, the only test case missing is `WS with kl=y (sends balancer bytes)`